NOTICE.txt

```diff
@@ -77,3 +77,12 @@ This product contains "ifaddrs-android.c" and "ifaddrs-android.h" from dxr.mozil
     * https://webrtc.googlesource.com/src/+/master/LICENSE
   * HOMEPAGE:
     * https://webrtc.googlesource.com/
+
+---
+
+This product contains a derivation of the Fabian Fett's 'Base64.swift'.
```

Comment:
Nit: not “the” Fabian Fett, just Fabian Fett. :wink:

Reply:
@Lukasa damn it copy and paste and not reading once more. sorry. fixed now.